### PR TITLE
Article 060: Handshake

### PR DESCRIPTION
**Pull request by Teifun2**
#### Length of text ####
:page_with_curl: short

####  Difficulty to read ####
:mouse:  light

####  Difficulty to understand #### 
:school_satchel:  easy

####  Type of text #### 

:bowtie:  informative

####  Type of header #### 
 

:boom:  catchy

### CODE REVIEW

**Comment by Teifun2** [comment]
**Plenum:**
Problem:
- private key 
- Protocol -> intention -> capabilites
Idea:
- Different greetings can imply different things
- Not all greetings are universal

**Comment by MoiraZuber** [comment]
Added the new image. Kept it close to the suggestion.

File: articles/article-060-handshake.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN TITLE -->
+# Introductions are universal
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In every society there is some sort of formal introduction when people meet each other for the first time. This is such a fundamental process that it even finds application in computers.
+Usually when different processes communicate with each other there also exists this introduction phase at the beginning. In computer science we call this the handshake. This is the time, where the different processes agree to communicate with each other and clearly state their protocol. Sometimes the handshake is also used to identify each other by means of private keys or passwords.
```
Comment (marcelluethi) [inline articles/article-060-handshake.md]: Is the sentence "Usually, when different process ...." needed? I find it better without it. What do you think?

Comment (Teifun2) [reply articles/article-060-handshake.md]: I agree. I removed the sentence and restructured the sentence.

**Comment by Teifun2** [comment]
> w image. Kept it close to the suggestion.

I like it very much. Thank you!

**Comment by noverney** [comment]
I kind of reworked most of what you written but you can use it as suggestions to rework certain lines. The first sentence is a bit clunky and could be pared down to what I have below. Also, I am not sure if you mean the protocol in the last line. I thought  a handshake was just the general send, receive, and acknowledge.  

When people meet each other for the first time, there is usually some sort of formal introduction. This fundamental process also applies to computer science and is called a handshake. During a handshake, different processes agree to communicate with each other based on a set of rules, known as a protocol. Sometimes the protocol involves identifying each other using private keys or passwords.

**Comment by Teifun2** [comment]
> I kind of reworked most of what you written but you can use it as suggestions to rework certain lines. The first sentence is a bit clunky and could be pared down to what I have below. Also, I am not sure if you mean the protocol in the last line. I thought a handshake was just the general send, receive, and acknowledge.
> 
> When people meet each other for the first time, there is usually some sort of formal introduction. This fundamental process also applies to computer science and is called a handshake. During a handshake, different processes agree to communicate with each other based on a set of rules, known as a protocol. Sometimes the protocol involves identifying each other using private keys or passwords.

I like this change and will update the text accordingly.

**Pull request merged**